crawlerr/server.py

```
@@ -283,45 +309,55 @@ async def get_server_info(ctx: Context) -> Dict[str, Any]:
 # Server lifecycle management will be handled by FastMCP automatically
 async def startup_checks():
     """Initialize services and perform startup checks."""
-    logger.info(f"Starting Crawlerr server v0.1.0")
-    logger.info(f"Debug mode: {settings.debug}, Production: {settings.production}")
+    logger.info(f"[bold]📋 Starting Crawlerr server v0.1.0[/bold]")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)